### PR TITLE
Get the latest approved FRR BGP doc from Azure sonic to project arlo

### PR DESCRIPTION
**Pull request by smahara**
Add new v6only  parameter for neighbor which specifies that BGP session be established on V6 link local address only.
Updating  show CLI commands based on the latest CLI used.

### CODE REVIEW

File: doc/mgmt/SONiC_Design_Doc_Unified_FRR_Mgmt_Interface.md [hunk 1]
```diff
@@ -1066,4 +1064,43 @@ This enhancement to FRR-BGP Unified management framework does not disrupt data p
 
 # 9 Unit Test
 Testing of the configuration changes is manual and BGP configurations present in CONFIG_DB are converted to FRR configurations using config gen utility with help of Jinja template.
+
+## 9.1 Test case for v6only:
+When v6 is enabled on an BGP neighbor (interface), BGP session is establised on link local address only and not the IP address specified for the interface.
+
+Test CASE 1 : Configure BGP neighbor.
```
Comment (prvattem) [inline doc/mgmt/SONiC_Design_Doc_Unified_FRR_Mgmt_Interface.md]: Pls mention that, it is a V4 neighbor.

File: doc/mgmt/SONiC_Design_Doc_Unified_FRR_Mgmt_Interface.md [hunk 2]
```diff
@@ -1066,4 +1064,43 @@ This enhancement to FRR-BGP Unified management framework does not disrupt data p
...
+|sonic(conf-if-Ethernet0)# exit| |
+|sonic(config)# router bgp 101| |
+|sonic(config-router-bgp)# neighbor interface Ethernet 0| |
+|sonic(config-router-bgp-neighbor)# address-family ipv4 unicast| |
+|sonic(config-router-bgp-neighbor-af)# activate| |
+|sonic(config-router-bgp-neighbor-af)# exit| |
+|sonic(config-router-bgp-neighbor)# remote-as external||
+|sonic(config-router-bgp-neighbor)# do show bgp ipv4 unicast neighbors|BGP session is establised on IPV4|
+
+TEST CASE 2
```
Comment (prvattem) [inline doc/mgmt/SONiC_Design_Doc_Unified_FRR_Mgmt_Interface.md]: Mention that, we are enabling v6only here.

File: doc/mgmt/SONiC_Design_Doc_Unified_FRR_Mgmt_Interface.md [hunk 3]
```diff
@@ -1066,4 +1064,43 @@ This enhancement to FRR-BGP Unified management framework does not disrupt data p
...
+
+|Command Executed | Result      |
+|:------------------|:-----------------|
+|sonic(config-router-bgp-neighbor)# v6only| BGP session is re-established on Link local|
+
+
+TEST CASE 3
```
Comment (prvattem) [inline doc/mgmt/SONiC_Design_Doc_Unified_FRR_Mgmt_Interface.md]: Mention that v6only is getting disabled.

File: doc/mgmt/SONiC_Design_Doc_Unified_FRR_Mgmt_Interface.md [hunk 4]
```diff
@@ -59,6 +59,7 @@
 | 0.2 | 10/30/2019  | Venkatesan Mahalingam | Config DB schema changes |
 | 0.3 | 11/20/2019  | Venkatesan Mahalingam | Added various fields in config DB |
 | 0.4 | 12/18/2019  | Venkatesan Mahalingam | Addressed the comments on error handling and method of testing |
```
Comment (venkatmahalingam) [inline doc/mgmt/SONiC_Design_Doc_Unified_FRR_Mgmt_Interface.md:61]: This doesnt look like a latest document, please check with @eddyatdell or @zhaozhenhong

**Review by venkatmahalingam — APPROVED** [review]

File: doc/mgmt/SONiC_Design_Doc_Unified_FRR_Mgmt_Interface.md (hunk 3 above)
Comment (venkatmahalingam) [reply doc/mgmt/SONiC_Design_Doc_Unified_FRR_Mgmt_Interface.md]: @prvattem Sucheta is bringing the latest HLD from the community as part of this PR, she'll create a new PR for v6only feature.

Comment (dks19) [reply doc/mgmt/SONiC_Design_Doc_Unified_FRR_Mgmt_Interface.md]: Can you send me the PR for v6only feature ?

**Pull request merged**